src/nvim/ui.c

```diff
@@ -323,11 +341,10 @@ static void set_highlight_args(int mask, HlAttrs *attrs)
   }
 
   attrs->bold = mask & HL_BOLD;
-  attrs->standout = mask & HL_STANDOUT;
   attrs->underline = mask & HL_UNDERLINE;
   attrs->undercurl = mask & HL_UNDERCURL;
   attrs->italic = mask & HL_ITALIC;
-  attrs->reverse = mask & HL_INVERSE;
+  attrs->reverse = mask & (HL_INVERSE | HL_STANDOUT);
```

Comment:
Standout is almost always the same as reverse, but it is a very terminal specific term. Also, I don't think it would make sense to ever pass both standout and reverse to a terminal, so I thought it would be better to have a single attribute and let TUI decide to use standout when reverse mode is not available.

That said, I think we should go in [this direction](https://github.com/neovim/neovim/issues/1767#issuecomment-68656372) and allow arbitrary key/value pairs in the highlight command, which would move knowledge about text attributes from nvim core to UIs